client/packages/core/src/Reactor.js

```diff
@@ -460,7 +460,11 @@ export default class Reactor {
         break;
       case "refresh-presence":
         const roomId = msg["room-id"];
-        this._setPresencePeers(roomId, msg.data);
+        if (msg["edits"]) {
```

Comment:
One issue I realize: what happens to old clients? 

As it stands, I _think_ if the client doesn't upgrade their version of the client SDK, they won't be able to handle the new `refresh-presence` events from the server. 

--- 

Potential solution: 

Clients currently send a `versions` map when they init: 

https://github.com/instantdb/instant/blob/main/client/packages/core/src/Reactor.js#L1056

This would be the first time we use it, but perhaps what we can do is: 
1. We detect if the client is on a version of Instant > x
2. In that case, we start sending the `refresh-presence` events with edits in them. 
  a. Aside, perhaps we send different kinds of events too -- something like `reset-presence` and 'patch-presence', to keep the paths separate, but I don't feel strongly about this idea

--- 

What do you think?